fink_science/hostless_detection/processor.py

```diff
@@ -0,0 +1,49 @@
+"""
+    Implementation of the paper:
+    ELEPHANT: ExtragaLactic alErt Pipeline for Hostless AstroNomical
+    Transients
+    https://arxiv.org/abs/2404.18165
+"""
+
+from pyspark.sql.functions import pandas_udf
+from pyspark.sql.types import FloatType
+import pandas as pd
+
+from run_pipeline import HostLessExtragalactic
+from pipeline_utils import load_json
+
+CONFIGS = load_json("./fink_science/hostless_detection/config.json")
+
+
+@pandas_udf(FloatType())
+def run_potential_hostless(
```

Comment:
As is, your module is not tested. You should a minima add a `__init__.py` in the folder, and add a unit test (using Spark) to test the relevance of the `pandas_udf`. Ideally, you should test against known alerts for which you know the results, so that we can anticipate any problem related to code change later. Let me know if you need help to set up a test.